src/utils/update/mapper.utils.ts

```diff
@@ -98,5 +98,9 @@ type AddressMatches = {
 };
 
 export const isSameAddress: AddressMatches = (address1: any, address2?: any) => {
-  return !address2 || Object.keys(address1).every(key => address1[key] === address2[key]);
```

Comment:
please check existing usage of this. I think it was saying address2 being undefined - defaults address2 to address1 so they are equal. So address2 null => the addresses are equal!

Reply:
I don't believe this was working as expected. The "yes" radio button was being selected despite having a service address -  change has fixed this